src/views/settings/SettingsNotifications.tsx

```diff
@@ -20,7 +38,31 @@ const SettingsNotifications: Screen<"SettingsNotifications"> = () => {
   const notifications = account.personalization.notifications;
 
   // Global state
-  const [enabled, setEnabled] = useState(notifications?.enabled || false);
+  const [enabled, setEnabled] = useState<boolean | null>(
+    notifications?.enabled || false
+  );
+  useEffect(() => {
+    const test = async () => {
```

Comment:
```suggestion
    const handleNotificationPermission = async () => {
```

Un nom plus clair serait bien